models/models.go

```diff
@@ -18,6 +18,8 @@ const (
 	StepRunStatusCodeSkipped = 3
 	// StepRunStatusCodeSkippedWithRunIf ...
 	StepRunStatusCodeSkippedWithRunIf = 4
+	// StepRunStatusCodePreparationFailed ...
```

Comment:
Introduced new state to be able to differentiate failed (but ran) and failed during configuration steps